renderer/helpers/filesystem.js

```diff
@@ -57,6 +57,19 @@ const readFile = filePath => new Promise((resolve, reject) => {
   });
 });
 
+const requireFile = filePath => new Promise((resolve, reject) => {
+  fs.readFile(filePath, 'utf8', (err, json) => {
+    if (err) { reject(err); }
+    else {
+      const colorSets = Function('return ' + json.replace('exports.colors = ', ''))()
```

Comment:
I think this is a great stop-gap for color sets that have been exported from themer. Let's add just one more safeguard in the event the user authors their own `colors.js` and uses a slightly different syntax (e.g., `module.exports.colors =`) and the import fails, by wrapping this in a try/catch and rejecting on error. (See for example some files that would fail: https://github.com/mjswensen/themer/blob/master/cli/packages/themer-colors-nova/lib/index.js or https://github.com/mjswensen/themer/blob/master/cli/packages/themer-colors-github-universe/lib/index.js)

Reply:
Good idea. I'll try look into this soon!